src/apis/api_v1.py

```
@@ -0,0 +1,220 @@
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)